### DISCUSSION TITLE
Are there any actions to resize the telescope window while its open?

### DISCUSSION THREAD

**Discussion by IndianBoy42** [nvim-telescope/telescope.nvim#1455] [Q&A] [answered]
I want to make some keybindings that can resize/move the telescope window while its open, as well as show/hide the preview window.

**Comment by l-kershaw** [comment, marked as answer]
This is currently close to being merged.
PR #1459 includes actions for toggling `preview`, `mirror` and `prompt_position`.
Custom actions for personal use will be able to be created in a similar way once this is merged.

**Reply by l-kershaw** [reply]
This has now been merged, an example of how to use it is available [here](https://github.com/nvim-telescope/telescope.nvim/wiki/Configuration-Recipes#add-mapping-to-toggle-the-preview).
